app/models/role.rb

```diff
@@ -16,6 +16,7 @@ class Role < ApplicationRecord
     trusted
     warned
     workshop_pass
+    forem_owner # The account that owns the Forem instance
```

Comment:
(**nitpick, non-blocking**):  I don't think this comment is necessary, because the role name is or will be self explanatory.

Reply:
Yeah, this was when it was called `forem_creator` initially. I'll remove it.